virttest/qemu_vm.py

```diff
@@ -2671,9 +2671,14 @@ def create(self, name=None, params=None, root_dir=None,
                 continue
             iso = cdrom_params.get("cdrom")
             if iso:
-                iso = utils_misc.get_path(data_dir.get_data_dir(), iso)
-                if not os.path.exists(iso):
+                if cdrom_params.get("enable_curl") == "yes":
+                    iso = storage.get_image_filename(cdrom_params, None)
+                else:
+                    iso = utils_misc.get_path(data_dir.get_data_dir(), iso)
```

Comment:
Suggest introducing this as a new function `get_iso_filename(params, root_dir, basename=False)` to the `storage` module and calling it at here.

Reply:
updated